### DISCUSSION TITLE
Can anyone give the proper documentation of plugin for EMQX version=>5.0

### DISCUSSION THREAD

**Discussion by sc-sityad** [emqx/emqx#10749] [Q&A]
Q1) I am facing error after download the directory of emqx template plugin. I am not able to perform **make rel** command due to following error.     
 `/home/sitaram/Desktop/Emqx/plugins/emqx-plugin-template/rebar3 emqx_plugrel tar
escript: exception error: undefined function rebar3:main/1
  in function  escript:run/2 (escript.erl, line 758)
  in call from escript:start/1 (escript.erl, line 277)
  in call from init:start_em/1 
  in call from init:do_boot/3 
make: *** [Makefile:49: rel] Error 127
`
I think its related to erlang/OTP 25 related or something else. 

Q2) Is there any easy documentation which elaborate how to update plugins.

**Comment by lafirest** [comment]
@sc-sityad 
By our other discussion, I think you have already fixed this problem, so I closed this.

**Comment by sc-sityad** [comment]
Its fine.

On Thu, May 25, 2023 at 12:45 PM lafirest ***@***.***> wrote:

> @sc-sityad <https://github.com/sc-sityad>
> By our other discussion, I think you have already fixed this problem, so I
> closed this.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/emqx/emqx/discussions/10749#discussioncomment-5997682>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2IBHJWDHMJI7UQ5FDO4E6LXH4BINANCNFSM6AAAAAAYHVLP74>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
The information contained in this e-mail is confidential and proprietary to 
Scry Analytics and it is intended only for use by the individual or entity 
to which it is addressed. If the reader of this message is not the intended 
recipient, you are hereby notified that any review, re-transmission, 
dissemination, distribution, copying or other use of, or taking of any 
action in reliance upon this information is strictly prohibited. If you 
have received this communication in error, please contact the sender and 
delete the material from your computer.
